File: AdotePetJavaProjetct/src/Foto.java
package adotepet;

public class Foto {
    
    private String link;
    private String descricao;
    
    public Foto(String link, String descricao){
        
        setLink(link);
        setDescricao(descricao);      
    }

    
    public String getLink() {
        return link;
    }

    public void setLink(String link) {
        this.link = link;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }
    
    
}

File: AdotePetWeb/backend/api/model/Foto.php
<?php

/**
 * @author Matheus
 */
class Foto {
   
    private $descricao; 


    public function get_descricao() {
        return $this->descricao;
    }
    
    public function set_descricao($descricao) {
        $this->descricao = $descricao;
        return $this;
    }
 
}
